po/POTFILES.in

```diff
@@ -1,5 +1,6 @@
 data/remotes.d/lvfs.metainfo.xml
 data/remotes.d/lvfs-testing.metainfo.xml
+libfwupdplugin/tests/bios-attrs/dell-xps13-9310/strings.txt
```

Comment:
should we check this txt file?
https://github.com/fwupd/fwupd/blob/main/contrib/ci/check-potfiles.py#L20

Reply:
thx, added it.